packages/backend/src/index.ts

```diff
@@ -8,12 +8,9 @@ export * from './tokens/verify';
 
 export type ClerkOptions = CreateBackendApiOptions & Pick<CreateAuthenticateRequestOptions, 'jwtKey'>;
 
-export default function Clerk(options: ClerkOptions) {
+export function Clerk(options: ClerkOptions) {
```

Comment:
🙃 This change should have been in a separate PR but it's OK for now :)

Reply:
You're right. I've cherry picked and extracted this into another PR because I need it before I can merge this one :)